test/RelayHubPenalizations.test.ts

```diff
@@ -40,7 +40,7 @@ contract('RelayHub Penalizations', function ([_, relayOwner, relayWorker, otherR
   before(async function () {
```

Comment:
Missing test for wrong gas limit parameter penalization... and penalizer change as well

Reply:
the penalizer itself is still missing.